app/actions/websocket.ts

```diff
@@ -1126,8 +1140,15 @@ function handleOpenDialogEvent(msg: WebSocketMessage) {
 }
 
 // Helpers
-export function notVisibleUsersActions(state: GlobalState): Array<GenericAction> {
-    const knownUsers = getKnownUsers(state);
+export async function notVisibleUsersActions(state: GlobalState): Promise<Array<GenericAction>> {
+    let knownUsers: Set<string>;
+    try {
+        const fetchResult = await Client4.getKnownUsers();
+        knownUsers = new Set(fetchResult);
+    } catch (err) {
+        return [];
+    }
+    knownUsers.add(getCurrentUserId(state));
```

Comment:
why are we including the current user profile ?

Reply:
The server is returning all the other users that you know, but not yourself, if you don't add yourself to the list, you can end up cleaning your own profile from the list of user profiles.